org.osgi.test.common/src/main/java/org/osgi/test/common/annotation/InjectService.java

```diff
@@ -88,4 +88,12 @@
 	 */
 	long timeout() default DEFAULT_TIMEOUT;
 
+	/**
+	 * Indicate require services objectClass filter is not set by the Parameter
+	 * or Field. (objectClass=*).
+	 *
+	 * @return The timeout.
+	 */
+	boolean anyObjectClass() default false;
```

Comment:
Like DS, it would be better to have a `Class<?> service() default Object.class;` element here. See <https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.component.html#service.component-anyservice>.